bpo-35537: Add setsid parameter to os.posix_spawn() and os.posix_spawnp(). (#11608)
python/cpython: Misc/NEWS.d/next/Library/2019-01-18-13-44-13.bpo-35537.R1lbTl.rst

> os.posix_spawn() and os.posix_spawn() now has a setsid parameter.

Reviewer: ```suggestion
:func:`os.posix_spawn` and :func:`os.posix_spawn` now have a *setsid* parameter.
```

Reviewer 2: The second function should be `os.posix_spawnp`.

> os.posix_spawn() and os.posix_spawnp() now has a setsid parameter.

Reviewer: I suggest the format:

```suggestion
:func:`os.posix_spawn` and :func:`os.posix_spawnp` now have a *setsid* parameter. 
```

(it's have, not has, no? English is not my first language, I'm not sure :-))
